### ISSUE TITLE
No images in emails

### ISSUE THREAD

**Issue by nbussman** [humhub/humhub#1102]
Hey there =) 

humhub runs great, but I can't get profile images in mails to work. When I look at
the mail source there s.th. like this 
```
<a href=3D"https://humhub.duckdns.org=
/index.php?r=3Duser/profile&guid=3D6289495b-6659-4429-9a09-060d916773ca">=

                                                            <img
     =
                                                           src=3D"https://h=
umhub.duckdns.org/uploads/profile_image/6289495b-6659-4429-9a09-060d916773c=
a.jpg?cacheId=3D0"
                                                      =
          width=3D"50"
                                                  =
              alt=3D""
                                                  =
              style=3D"max-width:50px; display:block !important; border-rad=
ius: 4px;"
                                                              =
  border=3D"0" hspace=3D"0" vspace=3D"0"/>
```

After removing linebreaks and "=" in urls, they work. But where are they come from anwhy is there a §D affter the equal sign?

thanks for your support!
cheer Nils

**Comment by Ardnived** [comment]
It's been 8 months almost. I'm also having this problem.

**Comment by luke-** [comment]
@Ardnived which e-mail client do you using?

Make sure the BaseUrl (Administration -> Basic) is valid.

**Comment by Ardnived** [comment]
I'm using Apple's "Mail" application.
BaseUrl is set as I would expect (http://domain.com)

However, I think I see the problem. When I look at the source of the email I received this is the html for the profile image.
```
<img src="/uploads/profile_image/3af163ed-2907-4a12-991c-d0490d3c7e54.jpg?m=3D1453830057" width="50" alt="" style="max-width:50px; display:block !important; border-radius: 4px;" border="0" hspace="0" vspace="0"/>
```

Looking at this, it seems that issue is that the image URL is relative. Aka it doesn't have the domain at the start.

**Comment by luke-** [comment]
Which HumHub version do you using?

Cannot reproduce this...

**Comment by Ardnived** [comment]
> Currently installed version: 1.0.0-beta.3

**Comment by luke-** [comment]
Really strange, does this affects all images in e-mails and links?
Did you changed anything e.g. in config files?
Can you test a another mail client?
How do you start the cron?

**Comment by Ardnived** [comment]
1) I'm only currently receiving notifications from the Mail plugin. The only image is the user avatar, consequently that's the only one which I see the issue for. All of the links in the email are fine.
2) We have not changed anything in the config files.
3) I've just tried with my domain registrar's mail client (MailEnable) and the same issue occurs.
4) Not sure exact what you mean. I set up cron just using crontab. Here's what is currently on my crontab.
```
crontab -l
30 * * * * /var/www/yuvashakti.com/humhub/protected/yii cron/hourly >/dev/null 2>&1
00 18 * * * /var/www/yuvashakti.com/humhub/protected/yii cron/daily >/dev/null 2>&1
```
The HumHub admin page reports that the cronjobs are running.
![screen shot 2016-02-25 at 5 47 02 pm](https://cloud.githubusercontent.com/assets/1362167/13340763/dbd25c22-dbe7-11e5-9b2b-4045e62a8319.png)
